File: Aula_03/c++/02.cpp
#include <iostream>

int maior(int a, int b, int c){
    int mr = a + ((a - b) * (-1 * (a < b)));
    return mr + ((mr - c) * (-1 * (mr < c)));
}

int main(){
    int a,b,c;
    std::cin >> a >> b >> c;
    int mr = maior(a,b,c);
    std::cout << mr << std::endl;
    return 0;
}

File: Aula_03/python/02.py
def maior (a,b,c):
    maior = a + ((a - b) * (-1 * (a < b)))
    return maior + ((maior - c) * (-1 * (maior < c)))

a = int(input())
b = int(input())
c = int(input())
print(maior(a,b,c))